### DISCUSSION TITLE
✅ 🚀 Add Transition Duration option everywhere + custom unit for that

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21295] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

Currently, the Transition Duration option is in some widgets. But it would be great if you add this option in every widget that has a hover mode. For example, add a button widget. In the Style tab, when you click on the Hover option, there will be a Transition Duration option. **The Transition Duration option should be added in each section that has a hover mode. This will be really necessary and useful.**
<hr>

Also, Transition Duration option should support custom unit. For example, we can write for Transition Duration :

transform 0.3s ease-out or Everything else

### Describe the solution you'd like

1- Add Transition Duration option for all widgets that have hover mode.

2- Support custom unit for Transition Duration

**Comment by rami-elementor** [comment]
Elementor 3.20 adds "Transition Duration" to 10 widgets. 1 on the free version and 9 on the pro version.
